app/experimenter/experiments/forms.py

```diff
@@ -47,10 +48,15 @@ def clean(self, value):
         cleaned_value = super().clean(value)
 
         if cleaned_value:
+            err_str = "Please Provide a Valid URL ex: {}show_bug.cgi?id=1234"
             if settings.BUGZILLA_HOST not in cleaned_value:
-                err_str = (
-                    "Please Provide a Valid URL ex: {}show_bug.cgi?id=1234"
+                raise forms.ValidationError(
+                    err_str.format(settings.BUGZILLA_HOST)
                 )
+            query = urlparse(cleaned_value).query
```

Comment:
Since this is basically the same as what we do in `get_bugzilla_id` I'd import that function and use it here instead of reimplementing it, and then just check if it's `None` and raise the ValidationError that way.